src/frontends/lean/token_table.cpp

```diff
@@ -96,7 +96,7 @@ void init_token_table(token_table & t) {
          {"@@", g_max_prec}, {"@", g_max_prec},
          {"sorry", g_max_prec}, {"+", g_plus_prec}, {"->", g_arrow_prec}, {"<-", 0},
          {"match", 0}, {"^.", g_max_prec+1},
-         {"renaming", 0}, {"extends", 0}, {nullptr, 0}};
+         {"renaming", 0}, {"extends", 0}, {"generalizing", 0}, {nullptr, 0}};
```

Comment:
It would probably be preferable if we could add new tactic "keyword arguments" without changing the global token table. For identifier-like keywords, we could do that with a scanner decorator (both before and after implementing lookahead).